src/linear_algebra/linear_independent.lean

```diff
@@ -462,7 +462,7 @@ begin
         simp only at hxy,
         rw hxy,
         exact (subset_span (mem_range_self y₂)) },
-      exact false.elim ((hindep x₁).ne_zero h0) } },
+      exact false.elim (linear_independent.ne_zero _ (hindep x₁) h0) } },
```

Comment:
We can still use dot notation, we just need to add an extra underscore:
```suggestion
      exact false.elim ((hindep x₁).ne_zero _ h0).elim } },
```

This should work for the changes in the other file as well.

Reply:
Interesting, I didn't realize that still worked I thought it'd look for `ne_zero` on `i`. Will give it a shot and update.

Reply:
Yep, Lean looks for the first explicit argument with a type equal to the namespace, see [Section 9.1 in TPiL](https://leanprover.github.io/theorem_proving_in_lean/structures_and_records.html#declaring-structures).

Reply:
Addressed I believe. Thanks for the tip!